LayoutTests/TestExpectations

```diff
@@ -7750,3 +7750,6 @@ webkit.org/b/286883 imported/w3c/web-platform-tests/css/CSS2/floats/remove-block
 webkit.org/b/286884 imported/w3c/web-platform-tests/css/CSS2/floats/zero-width-floats-positioning.tentative.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/CSS2/floats/float-nowrap-hyphen-rewind-1.html [ ImageOnlyFailure ]
 [ Debug ] imported/w3c/web-platform-tests/css/CSS2/floats/line-pushed-by-floats-crash.html [ Skip ]
+
+# Damage-related tests are only supported on GTK and WPE
+damage/ [ Skip ]
```

Comment:
Since damage are glib specific, I think they can be added to platform/glib/damage

Reply:
In the top most part of the expectations I mark them as skip, then in the glib expectations I mark them as pass. Is that not the way to do it?